src/functions/template-tags/general.php

```diff
@@ -1159,7 +1166,7 @@ function tribe_events_event_schedule_details( $event = null, $before = '', $afte
 		 * @param string $before    part of the HTML wrapper that was prepended
 		 * @param string $after     part of the HTML wrapper that was appended
 		 */
-		return apply_filters( 'tribe_events_event_schedule_details', $schedule, $event->ID, $before, $after );
+		return $cache_details[ $cache_details_key ] = apply_filters( 'tribe_events_event_schedule_details', $schedule, $event->ID, $before, $after );
```

Comment:
:no_entry_sign: Assignments must be the first block of code on a line

<sub>hash: bdc7d12f9935a6fd58a35618f530b000</sub>